### PR TITLE
Update tool output

### PR DESCRIPTION
**Pull request by martinmaly**

### CODE REVIEW

File: installer/pkg/cmd/gcp_broker.go [hunk 1]
```diff
@@ -142,14 +142,14 @@ func addGCPBroker() error {
 	if err != nil {
 		// Clean up the newly generated key if the command failed.
 		cleanupNewKey(brokerSAEmail, key)
-		return fmt.Errorf("error generating configs for GCP broker :: %v", err)
+		return fmt.Errorf("error generating configs for service broker :: %v", err)
```
Comment (maqiuyujoyce) [inline installer/pkg/cmd/gcp_broker.go]: So we are fine with both the "Service Broker" and "service broker"?

File: installer/pkg/cmd/gcp_broker.go [hunk 2]
```diff
@@ -221,7 +221,7 @@ func getOrCreateGCPServiceAccount(name, email string) error {
 	_, err := gcp.GetServiceAccount(email)
 	if err != nil {
 		// TODO(droot): distinguish between real error and NOT_FOUND error
-		err = gcp.CreateServiceAccount(name, "Service Catalog GCP Broker Service Account")
+		err = gcp.CreateServiceAccount(name, "Google Cloud Platform Service Broker")
```
Comment (maqiuyujoyce) [inline installer/pkg/cmd/gcp_broker.go]: I check the display name of the service accounts and found they always end with "service account".

File: installer/pkg/cmd/gcp_broker.go [hunk 3]
```diff
@@ -464,8 +464,8 @@ func NewCreateGCPBrokerCmd() *cobra.Command {
 	cfg := &createBrokerConfig{}
 	cmd := &cobra.Command{
 		Use:   "create-gcp-broker",
-		Short: "Create a GCP broker",
-		Long:  "Creates a GCP broker without adding it to an existing cluster",
+		Short: "Create the Service Broker",
+		Long:  "Creates a Google Cloud Platform Service without adding it to an existing cluster",
```
Comment (maqiuyujoyce) [inline installer/pkg/cmd/gcp_broker.go]: Google Cloud Platform Service Broker - "Broker" is missing here.

File: installer/pkg/cmd/gcp_broker.go (hunk 1 above)
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: So I switched all "title-like" strings to Service Broker and all regular outputs or errors to just "service broker"

File: installer/pkg/cmd/gcp_broker.go (hunk 3 above)
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: Done

File: installer/README.md [hunk 4]
```diff
@@ -120,7 +122,8 @@ make
 
 ## Tutorial
 
-Once you have Service Catalog installed and GCP broker added to the cluster.  Follow this [basic tutorial](/installer/service-catalog-pubsub-tutorial.md) to get started with Service Catalog.
+Once you have Service Catalog installed and the Service Broker added to the cluster.
+Follow this [basic tutorial](/installer/service-catalog-pubsub-tutorial.md) to get started with Service Catalog.
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/README.md]: done

File: installer/pkg/cmd/gcp_broker.go [hunk 5]
```diff
@@ -142,14 +142,14 @@ func addGCPBroker() error {
...
-		return fmt.Errorf("error generating configs for GCP broker :: %v", err)
+		return fmt.Errorf("error generating configs for the service broker :: %v", err)
 	}
 
 	err = deployConfigs(dir, gcpBrokerFileNames)
 	if err != nil {
 		// Clean up the newly generated key if the command failed.
 		cleanupNewKey(brokerSAEmail, key)
-		return fmt.Errorf("error deploying GCP broker configs :%v", err)
+		return fmt.Errorf("error deploying the service broker configs :%v", err)
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: done

File: installer/pkg/cmd/gcp_broker.go [hunk 6]
```diff
@@ -316,14 +316,14 @@ func cleanupNewKey(email, key string) {
...
+		Short: "Remove the Service Broker",
+		Long:  `Removes Google Cloud Platform Service Broker from service catalog`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := removeGCPBroker(); err != nil {
-				fmt.Println("failed to remove GCP broker")
+				fmt.Println("Failed to remove the service broker")
 				return err
 			}
-			fmt.Println("GCP broker removed successfully.")
+			fmt.Println("The service broker removed successfully.")
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: Done

File: installer/pkg/cmd/gcp_broker.go [hunk 7]
```diff
@@ -316,14 +316,14 @@ func cleanupNewKey(email, key string) {
...
 	return &cobra.Command{
 		Use:   "remove-gcp-broker",
-		Short: "Remove GCP broker",
-		Long:  `Removes a GCP broker from service catalog`,
+		Short: "Remove the Service Broker",
+		Long:  `Removes Google Cloud Platform Service Broker from service catalog`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := removeGCPBroker(); err != nil {
-				fmt.Println("failed to remove GCP broker")
+				fmt.Println("Failed to remove the service broker")
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: done

File: installer/pkg/cmd/gcp_broker.go [hunk 8]
```diff
@@ -341,7 +341,7 @@ func removeGCPBroker() error {
 	// remove GCP Broker k8s resources
 	err = generateConfigs(dir, gcpBrokerTemplateDir, gcpBrokerFileNames, nil)
 	if err != nil {
-		return fmt.Errorf("error generating configs for GCP :: %v", err)
+		return fmt.Errorf("error generating configs for the service broker :: %v", err)
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: done

File: installer/pkg/cmd/gcp_broker.go [hunk 9]
```diff
@@ -497,7 +497,7 @@ func createGCPBroker(cfg *createBrokerConfig) error {
 		msg = "Reused an existing"
 	}
 
-	fmt.Printf(`%s GCP Broker:
+	fmt.Printf(`%s service broker:
```
[the remark this answers is not in the document]
Comment (martinmaly) [reply installer/pkg/cmd/gcp_broker.go]: done

**Pull request merged**